src/containers/Article.js

```diff
@@ -285,6 +290,7 @@ Article.propTypes = {
   slugToFetch: PropTypes.string,
   releaseBranch: predefinedPropTypes.releaseBranch,
   isAuthed: PropTypes.bool,
+  userRole: PropTypes.array,
```

Comment:
Could we mark `userRole` isRequired here ?

Reply:
fixed